main/src/ca/uwaterloo/flix/api/lsp/provider/CompletionProvider.scala

```diff
@@ -131,6 +132,16 @@ object CompletionProvider {
       //
       case SyntacticContext.Import => ImportCompleter.getCompletions(context)
 
+      //
+      // Keywords. Note that this is not the context in which keywords can occur, but the context of a keyword, i.e. inside a keyword.
```

Comment:
Perhaps all context would benefit from such a description? And perhaps not here, but where the contexts are declared.

Reply:
Hmm. Yes. You might be right. I added this here because I sensed a danger of confusion due to there being a KeywordCompleter, which has no relation to SyntacticContext.Keyword. But you're right, I think it would be good to added such descriptions as docs for at the decl site

Reply:
OK, so lets move the comments to there and expand.